pkg/networkmanager/firewall.jsx

```diff
@@ -777,7 +800,39 @@ export class Firewall extends React.Component {
     }
 
     onRemoveZone(zone) {
-        firewall.deactiveateZone(zone);
+        let body;
+        if (firewall.zones[zone].services.indexOf("cockpit") !== -1)
+            body = <Alert type="warning">{_("This zone contains the cockpit service. Removing the zone risks making the web console unreachable.")}</Alert>;
```

Comment:
I don't think the use of the yellow box is quite right here.
Couldn't find any Patternfly4 guidelines about it, but this is what was recommended in Patternfly3
![mockup](https://www.patternfly.org/v3/pattern-library/communication/message-dialog/img/msg-dialog-withicon.png)

Reply:
So I would recommend to get rid of the background and use a bigger icon, to make the whole dialog look like a warning.

Reply:
Updated but still using PF